src/events/cd-event-list.vue

```diff
@@ -78,6 +94,25 @@
 </script>
 <style scoped lang="less">
   @import "../common/variables";
+  .join-button {
```

Comment:
We don't want the mixin to create a `join-button` class, so use `.join-button() {` instead. Parenthesis define it as a mixin only.